PrivateCollections/Germany/Neukirchen005.xml

```diff
@@ -141,98 +141,171 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                          in black frame.</desc>
+                        <q xml:lang="gez">ታሪክ᎓ ዘገባን</q>
                       </item>
+
                       <item xml:id="e7">
-                        <locus target="#160v"></locus>
-                        <desc>Erased text.</desc>
+                        <locus target="#14r"></locus>
+                        <desc>Written near the last verse of Ps. 16, in small fine script,
+                          in black frame.</desc>
+                        <q xml:lang="gez">ታሪክ᎓ ዘረፋይን</q>
+                      </item>
+
+                      <item xml:id="e8">
+                        <locus target="#86r"></locus>
+                        <desc>Directive as to how to use Ps. 91,written
+                          in small fine script, underlined.</desc>
+                        <q xml:lang="amh">ይኄይስ᎓ ተአምኖ᎓ እስከ᎓ ስምዓኒ᎓ ድረስ᎓ ድግም᎓
+                        መፍትሔ᎓ ሥራይ᎓ ተሐጸብ᎓ ኵለንታከ᎓ ፯ጊዕለት᎓</q>
                       </item>
 
+                      <item xml:id="e9">
+                        <locus target="#86r"></locus>
+                        <desc>The use of smaller script to accomodate longer lines,
+                        takedowns and takeups on some folios.</desc>
+
+                      </item>
+
+
                      </list>
                   </additions>
 
                   <bindingDesc>
                      <binding contemporary="true" xml:id="binding">
-                        <decoNote xml:id="b1">Two wooden boards of somewhat irregular size,
+                        <decoNote xml:id="b1" type="Boards">Two wooden boards of slightly irregular size,
                           cut not exactly along the grain but somewhat diagonally,
                           discolored, with traces of adhesives on the inner sides.</decoNote>
-                        <decoNote xml:id="b2" type="Cover">The boards are partly covered with
-                          untooled leather (quarter cover), with turn-ins
-                        held together by threads.</decoNote>
-                        <decoNote xml:id="b3">Four pairs of sewing stations. The sewing is loose,
-                          some quires are held together by means of a think thread of animal origin that goes
-                          through the leaves and the back board,
-                          and is knoted on the spine under the leather cover.                        
-
-                        held </decoNote>
-                          <decoNote xml:id="b4" type="bindingMaterial">
+                        <decoNote xml:id="b2" type="Cover">Untooled leather cover
+                          (quarter cover) that was glued to the boards (still holding on the
+                          back board); turn-ins are held by threads.</decoNote>
+                        <decoNote xml:id="b3" type="SewingStations">Two pairs of sewing stations. The sewing is damaged,
```

Comment:
this type normally only takes numbers as value

Reply:
(two pairs = 4 four pairs = 8)

Reply:
Done.